variable/technology/power-plant.yaml

```diff
@@ -0,0 +1,799 @@
+# this describes systems composed of one small reservoir and one plant with  pumping capacity that are used for short-term storage (eg week or day)
+Number of Units|Electricity|Hydro|Pumped Storage:
+   description: Number of units of kind "Pumped Storage "
+   unit:
+
+Maximum Active power|Electricity|Hydro|Pumped Storage:
+   description: Maximum active power of one  "Pumped Storage " unit
+   unit: MW
+
+Minimum Active power|Electricity|Hydro|Pumped Storage:
+   description: Minimum active power of one  "Pumped Storage" unit
+   unit: MW
+
+Maximum Storage|Electricity|Hydro|Pumped Storage:
+   description: Maximum volume of a Pumped Storage expressed in energy
+   unit: MWh
+
+Minimum Storage|Electricity|Hydro|Pumped Storage:
+   description: Minimum volume of a Pumped Storage expressed in energy
+   unit: MWh
+
+Pumping Efficiency|Electricity|Hydro|Pumped Storage:
+   description: efficiency of pumping for a Pumped Storage
+   unit: %
+
+Inertia|Electricity|Hydro|Pumped Storage:
+   description: inertia provided to the system by a Pumped Storage unit
+   unit: second
+
+Rate Frequency Containment Reserve|Electricity|Hydro|Pumped Storage:
+   description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
+                for a Pumped Storage unit
+   unit: %
+
+Rate Automatic Frequency Restoration Reserve|Electricity|Hydro|Reservoir:
+   description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
+                for a Pumped Storage unit
+   unit: %
+
+# description of simple hydrosystems -  run of river
+# this describes systems composed of one run of river plant (without storage)
+Number of Units|Electricity|Hydro|Run of River:
+   description: Number of units of kind "Run of River"
+   unit:
+
+Maximum Active power|Electricity|Hydro|Run of River:
+   description: Maximum active power of one  "Run of River" unit
+   unit: MW
+
+LoadFactor|Electricity|Hydro|Run of River|Profile:
+   description: Timeserie of hourly loadfactors for  "Run of River"
+   unit: %
+
+Minimum Active power|Electricity|Hydro|Run of River:
+   description: Minimum active power of one  "Run of River" unit
+   unit: MW
+
+Inertia|Electricity|Hydro|Reservoir:
+   description: inertia provided to the system by a typical "simple Hydro reservoir " unit
+   unit: second
+
+Rate Frequency Containment Reserve|Electricity|Hydro|Reservoir:
+   description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
+                for a typical "simple Hydro reservoir " unit
+   unit: %
+
+Rate Automatic Frequency Restoration Reserve|Electricity|Hydro|Reservoir:
+   description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
+                for a typical "simple Hydro reservoir " unit
+   unit: %
+
+# description of Energy Storage Systems
+Energy Charge Ratio|Electricity|Energy Storage System:
```

Comment:
still not clear to me what this means...

Reply:
I'm changing it to `Maximum Energy Charge` which is the upper bound that ESS could charge energy at the time "_t_".